scripts/scr_cheatcode/scr_cheatcode.gml

```diff
@@ -0,0 +1,156 @@
+function scr_cheatcode(argument0) //gml_Script_scr_cheatcode
+{
+    var _cheatcode = ""
+    var cheatcode_string = ""
+    var cheatcode_digits = ""
+    cheatcode_m_digits = ""
+    var neg = 0
+    if (argument0 != "")
+    {
+        cheatcode = argument0
+        _cheatcode = cheatcode
+        cheatcode_string = string_letters(cheatcode)
+        cheatcode_digits = string_digits(cheatcode)
+        cheatcode_string = string_lower(cheatcode_string)
+        if (cheatcode_digits != "")
+            cheatcode_digits = real(cheatcode_digits)
+        cheatcode_m_digits = cheatcode_digits
+        neg = string_count("-", cheatcode)
+        if (neg > 0)
+            cheatcode_m_digits = (cheatcode_digits * -1)
+        if (cheatcode_digits == "")
+        {
+            if (cheatcode_string == "infreq" && global.cheat_req == 0)
+            {
+                global.cheat_req = 1
+                cheatyface = 1
+                obj_controller.tempRequisition = obj_controller.requisition
+                obj_controller.requisition = 51234
+            }
+            else if (cheatcode_string == "infreq" && global.cheat_req == 1)
+            {
+                global.cheat_req = 0
+                cheatyface = 1
+                obj_controller.requisition = obj_controller.tempRequisition
+            }
+            else if (cheatcode_string == "infseed" && global.cheat_gene == 0)
+            {
+                global.cheat_gene = 1
+                cheatyface = 1
+                obj_controller.tempGene_seed = obj_controller.gene_seed
+                obj_controller.gene_seed = 9999
+            }
+            else if (cheatcode_string == "infseed" && global.cheat_gene == 1)
+            {
+                global.cheat_gene = 0
+                cheatyface = 1
+                obj_controller.gene_seed = obj_controller.tempGene_seed
+            }
+            else if (cheatcode_string == "debug" && global.cheat_debug == 0)
+            {
+                global.cheat_debug = 1
+                cheatyface = 1
+            }
+            else if (cheatcode_string == "debug" && global.cheat_debug == 1)
+            {
+                global.cheat_debug = 0
+                cheatyface = 1
+            }
+            if (cheatcode == "test")

```

Comment:
there should be an empty line above an if, to make it obvious that a new if-else clause chain has started.